packages/mds-audit-service/ormconfig.ts

```diff
@@ -17,4 +17,4 @@
 import { AuditRepository } from './repository'
 
 // Make connection options available to TypeORM CLI
-module.exports = AuditRepository.cli({ migrationsDir: 'repository/migrations' })
+export default AuditRepository.cli()
```

Comment:
The path to the migrations directory is now specified as part of the `migration:generate` CLI command, it's no longer an option.